### PR TITLE
New apps addition

### PR DESCRIPTION
**Pull request by shuvashish76**
Flashcard apps [**New**]
 - AnkiDroid
   - Anki Image Occlusion
   - AnkiEditor
 - ForgetMeNot

Keyboard
 - 8VIM 

Media Frontends
 - Twire (for Twitch)

Miscellaneous
 - Goodtime (Pomodoro Timer)
 - Vigilante (Privacy Indicator alternative)
 - dvd (download videos from multiple sources)
 - SpotiFlyer (downloaded music/songs from multiple sources)
 - SubHub (Subtitle downloader)
 - KDE Connect
 - (removed Feeder to RSS section)

Music
 - Dialog Music Player (minimal music player)
 - SongTube (currently available on [izzy-repo](https://apt.izzysoft.de/fdroid/index/apk/com.artxdev.songtube))
 
Notes
 - Quillnote
 - Logseq (available on [izzy](https://apt.izzysoft.de/fdroid/index/apk/com.logseq.app))

Office
 - OpenDocument

RSS Readers [**New**]
 - Feeder
 - NewsBlur

Science & Education [**New**]
 - freeCodeCamp (available on [izzy](https://apt.izzysoft.de/fdroid/index/apk/org.freecodecamp))

Sports & Health [**New**]
 - Red Moon (screenfilter)
 - HeartBeat (calculates heart rate from camera image)
 - Feeel (home workouts)
 - Paseo (step counting)

System
 - Insular (Shelter alternative
 - Scoop (stack trace of crashes)

Utilities
 - LTE Cleaner (system cleaner)

### CODE REVIEW

File: New apps addition [hunk 1]
```diff
@@ -303,10 +320,12 @@ Table of Contents
 - [**Markor**](https://github.com/gsantner/markor) <sup>**[[F-Droid](https://f-droid.org/app/net.gsantner.markor)]**</sup>
 - [**Notally**](https://github.com/OmGodse/Notally) <sup>**[[F-Droid](https://f-droid.org/app/com.omgodse.notally)]**</sup>
 - [**neutriNote**](https://github.com/appml/neutrinote) <sup>**[[F-Droid](https://f-droid.org/app/com.appmindlab.nano)]**</sup>
+- [**Logseq**](https://github.com/logseq/logseq)
```
Comment (offa) [inline New apps addition:323]: From a quick overview it looks like this contains proprietary google components.

**Review by offa — CHANGES_REQUESTED** [review]
Thank you! :+1: 

Minor editorial: Please use the `/app/` URL instead of `/packages/` for F-Droid.

File: New apps addition (hunk 1 above)
Comment (shuvashish76) [reply New apps addition:323]: Ok, removing it for now.

Request for F-Droid inclusion here: https://github.com/logseq/logseq/issues/4099
The developer seems interested, do give a thumbs up & subscriber issue thread for F-Droid related updates.

Comment (offa) [reply New apps addition:323]: :+1: already :-)

Comment (shuvashish76) [reply New apps addition]: Done

Comment (shuvashish76) [reply New apps addition]: Currently on [F-Droid](https://f-droid.org/packages/com.amaze.filemanager/) AmazeFileManager is too old.
The project is live & served on [izzy-repo](https://apt.izzysoft.de/fdroid/index/apk/com.amaze.filemanager).

IMO using old version is not good for privacy, shall we just remove the F-Droid link & keep source link?

Comment (offa) [reply New apps addition]: Sounds reasonable, yes!

**Comment by offa** [comment]
Is this PR ready to merge?

**Review by offa — APPROVED** [review]

**Comment by shuvashish76** [comment]
> Is this PR ready to merge?

Yes

**Comment by offa** [comment]
Thank you! :smile:

**Comment by offa** [comment]
No problem, I'll update the ToC anyway.

**Pull request merged**